code/go/0chain.net/chaincore/chain/entity.go

```diff
@@ -655,7 +655,8 @@ func (c *Chain) addInitialStakes(stakes []state.InitStake, balances cstate.State
 			ProviderID:   v.ProviderID,
 			DelegateID:   v.ClientID,
 			Balance:      v.Tokens,
-			RoundCreated: 0, // genesis round
+			RoundCreated: 0,
+			StakedAt:     gbCreationDate, // genesis round
```

Comment:
remove comment

Reply:
put the comment back to its original place